CHANGELOG.md

```diff
@@ -16,6 +16,7 @@
 - On X11, drag and drop now works reliably in release mode.
 - Added `WindowBuilderExt::with_resize_increments` and `WindowBuilderExt::with_base_size` to X11, allowing for more optional hints to be set.
 - Rework of the wayland backend, migrating it to use [Smithay's Client Toolkit](https://github.com/Smithay/client-toolkit).
+- Fix panic when trying to call `set_fullscreen(None)` on a window that has not been fullscreened prior.
```

Comment:
You need to specify which platform this is for.

Reply:
(...and don't forget to rebase.)

Reply:
Woops, my bad.